d3ploy/demand_driven_deployment_inst.py

```diff
@@ -135,13 +135,6 @@ class DemandDrivenDeploymentInst(Institution):
         default=0
     )
 
-    demand_std_dev = ts.Double(
```

Comment:
why don't we need this anymore?

Reply:
It's a double variable. It was in the code twice, I think from an old PR.

Reply:
It was defined twice